cpp/include/cudf/utilities/span.hpp

```diff
@@ -54,7 +54,7 @@ class span_base {
 
   static constexpr std::size_t extent = Extent;
 
-  constexpr span_base() noexcept : _data(nullptr), _size(0) {}
+  constexpr span_base() noexcept : _data(nullptr) {}
```

Comment:
Why was `_size` moved but not `_data`?

Reply:
The auto-fix for clang-tidy is far from perfect, I think that is probably the reason. I fixed it locally tho.